### DISCUSSION TITLE
Extra Documentation: Possibly?

### DISCUSSION THREAD

**Discussion by aditikilledar** [aditikilledar/dollar_bot_SE23#41] [General]
There seems to be a lot of redundant documentation, prime example being the files in here: https://github.com/aditikilledar/dollar_bot_SE23/tree/docu-merging-scattered-docs/python_docs.

I would like to remove this. Thoughts?

@shashank-madan @srina1h @aiyer786

**Comment by shashank-madan** [comment]
I agree. There is already documentation in place in the code files. This is redundant. We can remove this in my opinion. 
We should also see if we can modify the existing documentation to make it more succinct

**Reply by aditikilledar** [reply]
I'm glad you agree. I'll remove and merge with my next approved PR.
